### ISSUE TITLE
Revisions property to Workitem

### ISSUE THREAD

**Issue by gpe5280** [devopshq/tfs#15]
Add ability to retrieve revisions as a property to `Workitem`.

~~~~
@property
def revisions(self):
     return self.tfs.get_tfs_object('wit/workitems/{}/revisions'.format(self.id))
~~~~
I use this to build control charts.

**Comment by allburov** [comment]
Hello, thanks for your feedback, great idea.

Can you do the pull request from your own fork?

**Comment by allburov** [comment]
Will be fix in #16

**Comment by allburov** [comment]
Fixed in `dohq-tfs==1.0.54`
Also read more 
https://github.com/devopshq/tfs/blob/master/docs/ADVANCED.md
